cylc/flow/cfgspec/workflow.py

```diff
@@ -1628,7 +1629,7 @@ def get_script_common_text(this: str, example: Optional[str] = None):
 
                 This was done to allow users to control the order of
                 definition of the variables. This section will be removed
-                in Cylc 9.
+                in a later version of Cylc 8.
```

Comment:
```suggestion
                in a future version of Cylc 8.
```

(Might as well use the same term everywhere)